preprocessors/openstreetmap.schema.json

```diff
@@ -0,0 +1,138 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://image.a11y.mcgill.ca/preprocessors/openstreetmap.schema.json",
+    "type": "object",
+    "title": "OpenStreetMap Preprocessor",
+    "description": "Information on the contents of a place on a map",
+    "properties": {
+
+        "timestamp": {
+            "type": "number",
+            "minimum": 0
+        },
+        "request_uuid": {
+            "type": "string",
+            "pattern": "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-4[0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$"
+        },
+        "name": {
+            "type": "string",
+            "pattern": "^[A-Za-z]{2,6}(\\.((?!-)[A-Za-z0-9-]+(?<!-)))+$"
+        },
+        "data": {
+            "type": "object",
+            "properties": {
+                "point_of_interest": {
+                    "type": "array",
+                    "title": "point_of_interest",
+                    "description": "The collection of all points of interest within the specified area of the map",
+                    "items": {
+                        "type": "object",
+                        "title": "node" ,
+                        "properties": {
+                            "id": {
+                            "type": "number",
+                            "title": "node id",
+                            "description": "point of interest node id"
+                            },
+                            "lat": {
+                                "type": "number",
+                                "title": "Latitude",
+                                "description": "The latitude of the point of interest",
+                                "minimum": -90,
+                                "maximum": 90
+                            },
+                            "lon": {
+                                "type": "number",
+                                "title": "Longitude",
+                                "description": "The longitude of the point of interest",
+                                "minimum": -180,
+                                "maximum": 180
+                            },
+                            "cat": {
+                                "type": "string",
+                                "title": "Category",
+                                "description": "The category of the point of interest"
+                            },
+                            "name": {
+                                "type": "string",
+                                "title": "Name of point of interest",
+                                "description": "The specific name of the point of interest"
+                            }}}},
```

Comment:
I'd recommend against putting so many closing brackets on the same line. It can make it hard to read/edit in the future.